.buildkite/pipeline.yml

```diff
@@ -830,13 +830,13 @@ steps:
         command: "julia --color=yes --project=perf perf/flame.jl --job_id flame_perf_target_edmfx --target_job sphere_baroclinic_wave_rhoe_equilmoist_edmfx"
         artifact_paths: "flame_perf_target_edmfx/*"
         agents:
-          slurm_mem: 20GB
+          slurm_mem: 40GB
```

Comment:
Yikes, this is a bit much 😬

Reply:
Ohhhh, wait, @akshaysridhar, try changing the profiler back to a sample rate of 0.01!

Reply:
> Yikes, this is a bit much 😬

Agreed 😬